### PR TITLE
Disable edit claim label button when editing issues

### LINKED ISSUE #369: Automate periodic running of seam reports
Method TBD.

### PR DESCRIPTION
**Pull request by karinamzalez**
Resolves #369

### Description
- [x] disable Edit claim label button if changes are made to an issue 
- [x] add corresponding test for functionality in edit_ep_claim_labels_spec

### Acceptance Criteria
- [ x] Code compiles correctly

### CODE REVIEW

File: spec/feature/intake/edit_ep_claim_labels_spec.rb [hunk 1]
```diff
@@ -102,13 +102,32 @@
...
+
+      nr_label = Constants::EP_CLAIM_TYPES[nonrating_request_issue.end_product_establishment.code]["official_label"]
+      nr_row = page.find("tr", text: nr_label, match: :prefer_exact)
+
+      expect(page).to have_button("Save", disabled: true)
+      expect(nr_row).to have_button("Edit claim label", disabled: false)
+
+      # make issue update - remove issue or edit description
+      find("#issue-action-0").click
+      find("#issue-action-0_remove").click
```
Comment (meckila-navapbc) [inline spec/feature/intake/edit_ep_claim_labels_spec.rb:116]: It may be helpful to have another test case for when an issue is added here

**Comment by nanotone** [comment]
I'm going to merge `master` in, to try and figure out why Circle isn't running its checks again.

File: spec/feature/intake/edit_ep_claim_labels_spec.rb [hunk 2]
```diff
@@ -102,13 +102,55 @@
...
+        expect(nr_row).to have_button("Edit claim label", disabled: false)
+
+        # make issue update - add issue
+        find("#button-add-issue").click
+        find(".cf-select").click
+        find("#react-select-2-option-1").click
+        fill_in "decision-date", with: "08192020"
+        fill_in "Issue description", with: "this is a description"
+
+        find("#Add-issue-5-button-id-1").click
```
Comment (nanotone) [inline spec/feature/intake/edit_ep_claim_labels_spec.rb:139]: There are some neat shortcuts in Capybara for writing these that could make this section more like a user testing script and less reliant on ID selectors, e.g.

```
click_on("Add issue")
find(".cf-select", text: "Select or enter").click
find(".cf-select__option", text: "Unknown issue category").click
```

Maybe give that a try, but I don't think we follow any sort of style guide when writing feature specs so don't sweat it 😄

**Review by nanotone — APPROVED** [review]
Just the comment about feature test style, but this looks good to me!

Remember to set up a demo with product before moving this ticket into Done.

**Pull request merged**